src/Components/NavBar/NavBar.tsx

```diff
@@ -382,7 +382,14 @@ export const NavBar: React.FC = track(
                     >
                       <NavBarItemUnfocusableAnchor
                         href="/artists"
-                        onClick={handleClick}
+                        onClick={event => {
+                          handleClick(event)
+
+                          // Small timeout to avoid transition race condition
+                          setTimeout(() => {
+                            setVisible(false)
+                          }, 100)
```

Comment:
I couldn't make this work without a delay -- thoughts @dzucconi?

Reply:
Yeah, I mean we _could_ transition it using the web animations API and get a real callback. This is also fine. It's not exact in this case.